### PR TITLE
Bump crypto package and alpine image

### PR DESCRIPTION
**Pull request by thedadams**

### CODE REVIEW

**Review by jiaqiluo — APPROVED** [review]
LGTM

**Review by rawmind0 — APPROVED** [review]
LGTM

File: package/Dockerfile [hunk 1]
```diff
@@ -1,4 +1,4 @@
-FROM alpine:3.11.3
+FROM alpine:3.11.11
```
Comment (cbron) [inline package/Dockerfile:1]: Any reason not to push to 3.13 ? Just stability ? We'll probably migrate this later anyway so its fine.

**Review by cbron — APPROVED** [review]

File: package/Dockerfile (hunk 1 above)
Comment (thedadams) [reply package/Dockerfile:1]: Yes, stability. The issues were fixed in this patch, so I went with it.

**Pull request merged**